### PR TITLE
[QATM-3704] Connect to external HDFS

### PR DESCRIPTION
**Pull request by imorales-stratio**
@dcos
  Scenario: Test HDFS
    Given I get keytab 's000003-zktest.s000003' using deploy-api from path 's000003-zktest.s000003'
    Given I open a ssh tunnel to '10.130.19.106' with user '!{CLUSTER_SSH_USER}' using pem file '!{CLUSTER_SSH_PEM_PATH}' to host ip '172.16.2.165' host port '50400' to local port '50400' and save it with id 'hdfsTunnel'
    Given I open a ssh tunnel to '10.130.19.117' with user '!{CLUSTER_SSH_USER}' using pem file '!{CLUSTER_SSH_PEM_PATH}' to host ip '172.16.62.241' host port '50010' to local port '50010' and save it with id 'hdfsTunnel2'
    Given I open a ssh tunnel to '10.130.19.106' with user '!{CLUSTER_SSH_USER}' using pem file '!{CLUSTER_SSH_PEM_PATH}' to host ip '172.16.2.168' host port '50010' to local port '50011' and save it with id 'hdfsTunnel3'
    Given I open a ssh tunnel to '10.130.19.118' with user '!{CLUSTER_SSH_USER}' using pem file '!{CLUSTER_SSH_PEM_PATH}' to host ip '172.16.199.203' host port '50010' to local port '50012' and save it with id 'hdfsTunnel4'
    Given I open private connection to HDFS 's000003-hdfs1.s000003:50400' with principal 's000003-zktest.s000003' and keytab 'target/test-classes/s000003-zktest.s000003.keytab' and realm 'ECHO.HETZNER.STRATIO.COM' and kdc 'kerberos.echo.hetzner.stratio.com:88'
    Then I list content in HDFS directory '/tmp' and save it in environment variable 'hdfsCont'
    And I run 'echo !{hdfsCont}' locally
    Then I write local file 'target/test-classes/descriptor.json' to HDFS file '/tmp/descriptor6.json'
    Then I read file '/tmp/descriptor6.json' from HDFS to local file 'target/test-classes/pruebahdfs.json'
    Then I list files in HDFS directory '/tmp' and save it in environment variable 'hdfsCont'
    And I run 'echo !{hdfsCont}' locally
    Then file '/tmp/descriptor6.json' exists in HDFS
    Then I delete file '/tmp/descriptor6.json' from HDFS
    Then file '/tmp/descriptor6.json' does not exist in HDFS
    And I close HDFS connection
    And I close the ssh tunnel with id 'hdfsTunnel'
    And I close the ssh tunnel with id 'hdfsTunnel2'
    And I close the ssh tunnel with id 'hdfsTunnel3'
    And I close the ssh tunnel with id 'hdfsTunnel4'

  Scenario: Test HDFS
    Given I open connection to HDFS 'hdfs.epsilon.labs.stratio.com:8020' with principal 'sparktest' and keytab '/home/imorales/Descargas/sparktest.keytab' and realm 'REALM.DOM' and kdc 'public-agent03.golf.labs.stratio.com:88'
    Then I list content in HDFS directory '/tmp' and save it in environment variable 'hdfsCont'
    And I run 'echo !{hdfsCont}' locally
    Then I write local file 'target/test-classes/descriptor.json' to HDFS file '/tmp/descriptor2.json'
    Then I read file '/tmp/descriptor2.json' from HDFS to local file 'pruebahdfs.json'
    Then file '/tmp/descriptor2.json' exists in HDFS
    Then I delete file '/tmp/descriptor2.json' from HDFS
    Then file '/tmp/descriptor2.json' does not exist in HDFS
    And I close HDFS connection

### CODE REVIEW

File: src/main/java/com/stratio/qa/specs/HDFSSecSpec.java [hunk 1]
```diff
@@ -64,6 +64,20 @@ public void openHDFSConnection(String hdfsHost, String coreSite, String hdfsSite
...
+    /**
+     * Open connection to HDFS cluster
+     *
+     * @param hdfsHost          brokers URL to connect to cluster
+     * @param keytabPath        keytab file path
+     * @param realm             kerberos realm
+     * @param principal         User to connect
+     * @param kdc               kerberos kdc
+     */
+    @Given("^I open( privacy)? connection to HDFS '(.+?)' with principal '(.+?)' and keytab '(.+?)' and realm '(.+?)' and kdc '(.+?)'$")
```
Comment (aalfonso-stratio) [inline src/main/java/com/stratio/qa/specs/HDFSSecSpec.java:76]: No quedaría mejor: @Given("^I open( private)? connection to HDFS...

**Review by aalfonso-stratio — CHANGES_REQUESTED** [review]
mira el comentario

File: src/main/java/com/stratio/qa/specs/HDFSSecSpec.java (hunk 1 above)
Comment (imorales-stratio) [reply src/main/java/com/stratio/qa/specs/HDFSSecSpec.java:76]: Correcto, actualizo

**Comment by imorales-stratio** [comment]
@aalfonso-stratio ya lo deje corregido

**Review by aalfonso-stratio — APPROVED** [review]

**Pull request merged**